### PR TITLE
Carousel implementation

### LINKED ISSUE #6: Who We Are > Photos Carousel
Automated photo carousel within the "Who We Are" section

Screen Shot 2021-07-15 at 10 44 41 PM

### PR DESCRIPTION
**Pull request by dchen150**
screenrecording

## Description
Resolves #6. Carousel component. Takes a list of images as the input and the width/height of the carousel. Can scroll through images using arrows. Has diamonds to represent which image is currently shown.

## Other considerations
colors aren't right right now

### CODE REVIEW

**Review by ianmah — CHANGES_REQUESTED** [review]
⚠⚠⚠⚠ayo why we have package-lock and yarn lock ⚠⚠⚠⚠

**Comment by ianmah** [comment]
Let's use this so we support any size of image (and dont stretch it out) 
https://www.w3schools.com/css/css3_object-fit.asp

**Comment by anneguo3** [comment]
Can this also auto-scroll without needing to click through the photos? 👀

**Comment by dchen150** [comment]
> Can this also auto-scroll without needing to click through the photos? 👀

auto-scrolls every 5 seconds of not being manually scrolled now. Also added object-fit: cover for the image css
screenrecording

File: components/Carousel.js [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+                            })}
+                        </FlexBox>
+                    </Container>
+                ) : (
+                    null
+                )
+            }
+        </>
+    )
+}
```
Comment (kevin-zou) [inline components/Carousel.js:103]: Nit: newline at end of file (we should probably get a linter set up soon)

Comment (kevin-zou) [inline components/Carousel.js:103]: Nit: newline at end of file (we should probably get a linter set up soon)

File: components/Carousel.js [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+                        <RightArrow onClick={() => setImageIndex(imageIndex == numImages - 1 ? 0 : imageIndex + 1)}/>
+                        <LeftArrow onClick={() => setImageIndex(imageIndex == 0 ? numImages - 1 : imageIndex - 1)}/>
+                        <Image src={images[imageIndex]} width={width} height={height}/>
+                        <FlexBox>
+                            {images.map((item, index) => {
+                                return index == imageIndex
+                                    ? <FilledDiamond key={index} />
+                                    : <EmptyDiamond key={index} />
+                            })}
+                        </FlexBox>
```
Comment (kevin-zou) [inline components/Carousel.js:95]: Can we center this under the image, as per the design?
image

**Review by kevin-zou — COMMENTED** [review]
Couple of comments, but looking good! 🚀

**Comment by dchen150** [comment]
image
 diamonds centered and new line added

**Review by kevin-zou — APPROVED** [review]
🔥

File: components/Carousel.js [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+                    </Container>
+                ) : (
+                    null
+                )
```
Comment (kozr) [inline components/Carousel.js:102]: Probably didn't need the ternary and could've just used `&&`

**Pull request merged**